src/managednetworkfabric/azext_managednetworkfabric/aaz/latest/nf/acl/_create.py

```diff
@@ -0,0 +1,384 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "nf acl create",
+)
+class Create(AAZCommand):
+    """Create a Access Control List resource
+
+    :example: Creates a Access Control List resource.
+        az nf acl create --resource-group "example-rg" --location "westus3" --resource-name "example-accesscontrollist" --address-family "ipv4" --conditions '[{"sequenceNumber": 123, "action": "allow", "destinationAddress": "1.1.10.10", "destinationPort": "1254", "sourceAddress": "1.1.1.0/24", "sourcePort": "1276", "protocol": "255", "annotation": "asdf"}]'
```

Comment:
You should use [shorthand syntax](https://github.com/Azure/azure-cli/blob/dev/doc/shorthand_syntax.md) in the examples so that it can cross shells.
```suggestion
        az nf acl create --resource-group "example-rg" --location "westus3" --resource-name "example-accesscontrollist" --address-family "ipv4" --conditions "[{sequenceNumber:123,action:allow, destinationAddress:'1.1.10.10',destinationPort:1254,sourceAddress:'1.1.1.0/24',sourcePort:1276,protocol:255,annotation:asdf}]"
```

Reply:
updated